message_queue_server/message_queue_service.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MESSAGE_QUEUE_SERVICE_H
+#define MESSAGE_QUEUE_SERVICE_H
+
+#include <grpcpp/grpcpp.h>
```

Comment:
For client we are hiding the external libraries in the header. Why not in server?

Reply:
I guess this is okay since we will be creating server executable.